spec/models/post_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe 'Post Model Tests' do
+  user = User.create(Name: 'Tom', Photo: 'https://unsplash.com/photos/F_-0BxGuVvo', Bio: 'Teacher from Mexico.')
+  post = Post.create(author: user, Title: 'Hello', Text: 'This is my first post')
+  comment = Comment.create(user:, post:, Text: 'Hi Tom!')
+  like = Like.create(user:, post:)
+  it 'should be a vaild post' do
+    expect(post).to be_valid
+  end
+
+  it 'should be not a vaild post if there is no a title' do
+    post.Title = ''
+    expect(post).to_not be_valid
+    post.Title = 'Hello'
```

Comment:
- [ ] Please write test statement under this declaration to be sure it's valid with correct values.